extension/js/common/core/mime.ts

```diff
@@ -111,20 +114,16 @@ export class Mime {
     return b.type === 'plainAtt' && b.attMeta && b.attMeta.inline && b.attMeta.type && ['image/jpeg', 'image/jpg', 'image/bmp', 'image/png', 'image/svg+xml'].includes(b.attMeta.type);
   }
 
-  private static headersToFrom = (parsedMimeMsg: MimeContent) => {
-    const headerTo: string[] = [];
-    let headerFrom;
-    if (Array.isArray(parsedMimeMsg.headers.from) && parsedMimeMsg.headers.from[0] && parsedMimeMsg.headers.from[0].address) {
-      headerFrom = parsedMimeMsg.headers.from[0].address;
-    }
-    if (Array.isArray(parsedMimeMsg.headers.to)) {
-      for (const to of parsedMimeMsg.headers.to) {
-        if (to.address) {
-          headerTo.push(String(to.address));
+  private static headerGetAddress = (parsedMimeMsg: MimeContent, headersNames: Array<SendingType | 'from'>) => {
+    const result: { from: string[], to: string[], cc: string[], bcc: string[] } = { from: [], to: [], cc: [], bcc: [] };
```

Comment:
Why would from be an array?

Reply:
I agree, it would be better to have `from` like `string | undefined`.

I wanted to have all headers the same type so I can work with them easier